charts/atlantis/templates/statefulset.yaml

```diff
@@ -69,6 +69,16 @@ spec:
         secret:
           secretName: {{ template "atlantis.awsSecretName" . }}
       {{- end }}
+      {{- if .Values.githubApp }}
+      {{- if index .Values.githubApp "key.pem"}}
+      - name: github-app-key-volume
+        secret:
+          secretName: {{ template "atlantis.vcsSecretName" . }}
```

Comment:
needs `valueFrom`

Reply:
This is not true for volumes. `valueFrom` is only needed for env vars.